static/sass/config/_utilities.scss

```diff
@@ -0,0 +1,66 @@
+// Utilities
+// =========
+
+
+// Text Outline
+// ------------
+/// Outline text with a 1px solid shadow of any color,
+/// because CSS doesn't provide `spread-radius` on `text-shadow`.
+/// This is primarily used for improved link-underlines.
+/// @group typography
+/// @parameter {Color | String} $background ['background'] -
+///   Color (or Accoutrement color-key) to use for the outline.
+/// @link https://eager.io/blog/smarter-link-underlines/
+///   Smarter Text Underlines
+/// @see text-underline
+/// @example njk -
+///   <a href="#">link outline (decenders: Qhghjhqhy)</a>
+@mixin text-outline(
```

Comment:
Yeah. Basically, the size of the outline is determined by a combination of `$count` and `$spacing` — where spacing controls the offset of each individual shadow (basically the resolution) and count controls the number of shadows to create at that size. So `0.02em` spacing and `4` count should give us a total of `0.08em` outline...

hmmmm, seems like I could make those variables more clear...